### PR TITLE
adding wildcard example to header doc page

### PR DESCRIPTION
**Pull request by SaraOgaz**
I couldn't find an example usage of this on the doc pages, thought it would be useful to have here.

### CODE REVIEW

File: docs/io/fits/usage/headers.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Value Access, Updating, and Creating
 """"""""""""""""""""""""""""""""""""
 
 As shown in the :ref:`Getting Started <tutorial>` tutorial, keyword values can
-be accessed via keyword name or index of an HDU's header attribute. Here is a
+be accessed via keyword name or index of an HDU's header attribute. You can also use the wildcard character ``*`` to get the keyword value pairs that match your search string. Here is a
```
Comment (pllim) [inline docs/io/fits/usage/headers.rst]: Even if RST does not get PEP8 check, I would recommend the line not to be too long. I like the 79 char limit but some others might prefer 100.

Comment (mhvk) [inline docs/io/fits/usage/headers.rst]: Would be good to keep line length < 80 here.

**Comment by pllim** [comment]
Actually for this PR, I would like the CIs to run so we can be sure `doctest` likes the change. Can you please use `git commit --amend` to remove "[skip ci]" and push it again? Thanks.

File: docs/io/fits/usage/headers.rst (hunk 1 above)
Comment (pllim) [inline docs/io/fits/usage/headers.rst]: Hello, fellow Emacs user! :hand:

**Comment by mhvk** [comment]
Nice, one of the nice things of headers, so good to advertise.  Very minimal comment, maybe correct that as you push it for testing (it should indeed be tested, for the reasons @pllim mentioned just now).

**Comment by SaraOgaz** [comment]
@mvhk Should be fixed now.

**Comment by mhvk** [comment]
@pllim - I'll change to 1.3 milestone since this is not a bug fix (and thus means a bit less work...)

Otherwise, all looks good, so merging. Thanks, @SaraOgaz!

**Pull request merged**